backend/tests/cron.test.ts

```
@@ -0,0 +1,36 @@
+import { assertEquals } from 'testing/asserts.ts'
+import { describe, it } from 'testing/bdd.ts'
+import { FakeTime } from 'testing/time.ts'
+import { invokeBroadcastCron } from '../scheduledcron/cron.ts'
+
+describe(
+  'Invoke broadcast cron',
+  { sanitizeOps: false, sanitizeResources: false },
+  () => {
+    it.only('creates correct cron job format', () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)